doc/codedoc.rst

```diff
@@ -0,0 +1,13 @@
+==================
+Code documentation
```

Comment:
Would it make sense to have a few words here like "here are the docstrings for the main classes that users will see..." before jumping straight into the docstrings?